src/main/java/games/strategy/engine/framework/ui/NewGameChooserModel.java

```diff
@@ -55,9 +55,9 @@ public NewGameChooserEntry get(final int i) {
   }
 
   private static List<File> allMapFiles() {
-    final List<File> rVal = new ArrayList<>();
-    rVal.addAll(safeListFiles(ClientFileSystemHelper.getUserMapsFolder()));
-    return rVal;
+    final List<File> files = new ArrayList<>();
+    files.addAll(safeListFiles(ClientFileSystemHelper.getUserMapsFolder()));
+    return files;
```

Comment:
Instead of using addAll the collection could be passed to the constructor of ArrayList, and thr constructor could be put after the return statement

Reply:
Done in 1715bd2.